docs/admin/install/eventing/install-eventing-with-yaml.md

```diff
@@ -115,7 +113,7 @@ Follow the procedure for the broker of your choice:
         kubectl apply -f {{ artifact(org="knative-sandbox",repo="eventing-kafka-broker",file="eventing-kafka-broker.yaml")}}
         ```
 
-    For more information, see the [Kafka broker](../../../../eventing/broker/kafka-broker/) documentation.
+    For more information, see the [Kafka broker](../../../eventing/broker/kafka-broker/README.md) documentation.
```

Comment:
This link seems to just glitch for me in the preview?
https://deploy-preview-3948--dev-knative.netlify.app/development/eventing/broker/kafka-broker/

Reply:
good catch! Fixed this just now